tests/components/home_connect/test_init.py

```diff
@@ -122,85 +209,91 @@ async def test_setup(
     assert config_entry.state == ConfigEntryState.NOT_LOADED
 
 
-@patch.object(ConfigEntryAuth, "get_devices")
 async def test_update_throttle(
-    get_devices,
+    appliance: Mock,
     hass: HomeAssistant,
     config_entry: MockConfigEntry,
     integration_setup: Callable[[], Awaitable[bool]],
     setup_credentials: None,
     platforms: list[Platform],
+    get_appliances: MagicMock,
 ) -> None:
     """Test to check Throttle functionality."""
     assert config_entry.state == ConfigEntryState.NOT_LOADED
 
     assert await integration_setup()
-    assert get_devices.call_count == 0
     assert config_entry.state == ConfigEntryState.LOADED
+    assert not get_appliances.called
```

Comment:
I prefer using `Mock.call_count` on mocks that aren't specced to avoid the risk of making a typo and accessing the wrong attribute, which may not be caught by the test since mocks will by default create attributes on access.